crates/holochain/src/conductor/handle.rs

```diff
@@ -477,6 +475,7 @@ pub struct ConductorHandleImpl<DS: DnaStore + 'static> {
         Arc<parking_lot::Mutex<HashMap<Arc<KitsuneSpace>, DbWrite<DbKindP2pAgentStore>>>>,
 
     /// The database for storing p2p MetricDatum(s)
+    #[allow(dead_code)]
     pub(super) p2p_metrics_env:
         Arc<parking_lot::Mutex<HashMap<Arc<KitsuneSpace>, DbWrite<DbKindP2pMetrics>>>>,
```

Comment:
Can this be removed?

Reply:
I'm going to be using this in the next follow-on PR, I'll remove the dead_code marker at that time.